### PR TITLE
Adds ubiquity for hard disk installation

### LINKED ISSUE #185: Enable hard disc install of Meilix
- Issue type: Feature request 

### Short description
We require one `Install the OS` option during the first screen of the ISO boot (that was removed and the ubiquity-enabled option to actually install the OS on a hard drive)

### PR DESCRIPTION
**Pull request by abishekvashok**
### Short description
I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

## Description
Adds ubiquity to be installed via /scripts/chroot.sh in the chroot env
so that it will be present on the live cd. Modifies loopback.cfg of
grub so that `Install Meilix` becomes visible which runs live
installation along `--only-ubiquity` parameter which fires up ubiquity.

Adds slides which will be displayed by ubiquity as well the text displayed
during the installation procedure to the `/ubiquity-slidehsow/` folder
which overrides the files in `/usr/share/ubiquity-slideshow/` to display
slides specific to meilix. This change is reflected in build.sh

Fixes #185 #144 

@fossasia/linux-devops-admin please review

### CODE REVIEW

**Comment by abishekvashok** [comment]
I coulld'nt relate this to a GCI task!

File: build.sh [hunk 1]
```diff
@@ -73,6 +73,8 @@ sudo cp -v skype-ubuntu_*_i386.deb chroot
 sudo cp -v meilix-imclient_*_all.deb chroot
 #SUSI deb builded by adding susi script
 sudo cp -v SUSI-desktop.deb chroot
+# ubiquity-slideshow slides
+sudo cp -v ubiquity-slideshow chroot/usr/share/
```
Comment (tabesin) [inline build.sh]: You may want to add an -r option

File: scripts/chroot.sh [hunk 2]
```diff
@@ -32,6 +32,9 @@ apt-get install -f
 # Install base packages
 apt-get -qq -y install xorg sddm lxqt
 
+# Install ubiquity
+apt-get -qq -y install ubiquity ubiquity-casper ubiquity-slideshow-ubuntu ubiquity-frontend-kde
```
Comment (tabesin) [inline scripts/chroot.sh]: why ubiquity-frontend-kde?

Comment (abishekvashok) [reply scripts/chroot.sh]: it's qt based.

**Comment by abishekvashok** [comment]
@tabesin done. I was sorry last time, also the position of theat copy command to override the contents have now been changed. Please do check :)

**Comment by abishekvashok** [comment]
Ping @fossasia/linux-devops-admins

**Comment by tabesin** [comment]
Here we go.

**Comment by abishekvashok** [comment]
Tq @tabesin should I leave the rest of the issues for others as there seems to be only 1 task in gci and much more issues labelled gci here. I don't have an issue to relate this with so I guess this won't earn a point for anyone. Am i blocking their points?

**Comment by tabesin** [comment]
We could add issues to GCI. I could add issues, it is only that the org admin has to approve them.

**Comment by abishekvashok** [comment]
Interesting...

**Comment by xeon-zolt** [comment]
@abishekvashok can you also test the installer? and add screenshots

**Comment by abishekvashok** [comment]
@xeon-zolt sure

**Comment by meets2tarun** [comment]
We once need to check the iso by downloading and booting it!

**Comment by abishekvashok** [comment]
@meets2tarun let me try.

**Pull request merged**